brozzler/ydl.py

```diff
@@ -418,6 +420,43 @@ def _try_youtube_dl(worker, ydl, site, page):
     return ie_result
 
 
+def get_video_captures(site, source="youtube"):
+    if not VIDEO_DATA_SOURCE:
+        return None
+
+    if VIDEO_DATA_SOURCE and VIDEO_DATA_SOURCE.startswith("postgresql"):
+        import psycopg
+
+        account_id = site.account_id if site.account_id else None
+        seed = site.metadata.ait_seed_id if site.metadata.ait_seed_id else None
+        if source == "youtube":
+            containing_page_url_pattern = "http://youtube.com/watch"  # yes, video data canonicalization uses "http"
```

Comment:
Does psycopg implicitly add "%" to the end of a LIKE pattern? I would expect something like "http://youtube.com/watch/%". This should be covered by a test.